x-pack/test/api_integration/apis/telemetry/management_collection.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as ts from 'typescript';
+import * as path from 'path';
+import expect from '@kbn/expect';
+import { getInterfacesDescriptors, loadProgram } from '@kbn/telemetry-tools';
+import { FtrProviderContext } from '../../ftr_provider_context';
+// @ts-ignore
```

Comment:
Can we remove this? Or is it needed for any TS check?

Reply:
Hmm i think the linter moved it around. I'll fix it